python_scripts/database_builder_v2.py

```diff
@@ -630,4 +599,4 @@ def rebuild_database(db_name, rebuild_db, delete_underlying_data=True):
     # Joined rooms
     glue_client.create_table(**schema_joined_rooms)
 
-    rebuild_all_s3_data_from_raw()
+    #rebuild_all_s3_data_from_raw()
```

Comment:
Why is this commented out now?

Reply:
Same as above, I was only wanting to rename a database using the code that was already there, didn't think there was any benefit to recreating the data as it hadn't moved.